assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -281,6 +282,9 @@ function MediaPane(props) {
             </MainButton>
           </FilterArea>
         </Padding>
+        <Feature name="sampleFlag">
+          <Message>{'This is a sample flag result'}</Message>
```

Comment:
Before merge, this should probably be removed and instead added as an example in a new `/docs/feature-flags.md` document.

Reply:
Done.